src/mbgl/tile/tile_cache.cpp

```diff
@@ -51,9 +51,15 @@ void TileCache::deferredRelease(std::unique_ptr<Tile>&& tile) {
     // If this temporary outlives the `schedule` call, and the function is executed immediately
     // by a waiting thread and is already complete, that temporary reference ends up being the
     // last one and the destruction actually occurs here on this thread.
-    std::function<void()> func{[tile_{CaptureWrapper<Tile>{std::move(tile)}}]() {
+    std::function<void()> func{[tile_{CaptureWrapper<Tile>{std::move(tile)}}, this]() mutable {
+        std::exchange(tile_.item, {});
```

Comment:
Why not `tile_.item = {}`?

Reply:
I was debugging before I remembered I needed to make this lambda mutable.